src/docs/sdk/performance/span-operations.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: "Span Operations"
+---
+
+Span operations are a short code identifying the type of operation the span is measuring. Span operations are low cardinality attributes - they should be as general as possible while still being human readable and useful.
+
+Operations are expected to follow [OpenTelemetry's semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/README.md) as much as possible.
+
+Below is a table containing a list of span operations that are recommended to be used.
+
+Operations are expected to generally take the form: `CATEGORY.ACTION`, where the categories and actions can have optional identifiers. For example, a redis database operation takes the form `db.sql.query`, where `db` is the category, `sql` is the category identifier, and `query` is the action. Another example is a mobile app rendering it's screen. It would have the operation `ui.load.hot`, where `ui` is the category, `load` is the action, and `hot` is the action identifier.
+
+It's important to keep categories consistent between sdks and integrations as they are used by Sentry in the [Operations Breakdown](https://docs.sentry.io/product/sentry-basics/tracing/event-detail/#operations-breakdown) feature. For example, both `db.init` and `db.query` will be categorized as database operations. The default operations breakdown config can be seen [here](https://github.com/getsentry/sentry/blob/809b7fe54c6f06cc1e4c503cf83ded896472a011/src/sentry/projectoptions/defaults.py#L74).
+
+Having both a category and action is recommended. Operations should be lowercased.
+
+# List of Operations
+
+## Browser
+
+| Category    | Usage           | Description                                                                                                         |
+| ----------- | --------------- | ------------------------------------------------------------------------------------------------------------------- |
+| pageload    |                 | A full page load of a web application                                                                               |
```

Comment:
Should we specify what should go into the description of the spans as well? I think it would be awesome to specify them precisely, so these are consistent as well.

Reply:
Yes I think this is a great idea, but out of the scope of this PR. I want to focus on getting the operations right, we can then take a look at descriptions next.

Reply:
I created an issue for that https://github.com/getsentry/develop/issues/463